Automattic/jetpack
# VideoPress: add Logo component

This PR adds the VideoPress Logo component.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: add Logo component

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Use StoryBook
* Confirm you see the VideoPress logo

<img width="627" alt="image" src="https://user-images.githubusercontent.com/77539/186970188-cc76eb9c-0c32-45eb-8235-5218862ec45b.png">
